External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -41,6 +42,10 @@ public class ASGenerator
         static private MemberModel contextMember;
         static private bool firstVar;
 
+        static List<ICompletionListItem> known;
+
+        public static List<ICompletionListItem> KnownList => known;

```

Comment:
However it requires you having VS2015, which is not possible for everyone out there. The same as I said for using the null conditional operator.